WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2248,6 +2251,27 @@ static GSourceFuncs _handlerIntervention =
                 reinterpret_cast<GCallback>(wpeNotifyWPEFrameworkMessageReceivedCallback), this);
             webkit_user_content_manager_register_script_message_handler(userContentManager, "wpeNotifyWPEFramework");
 
+            if (_config.UserScripts.IsSet()) {
+                auto loadScript =
+                    [&userContentManager](const std::string& path) {
+                        gchar* scriptContent;
+                        auto success = g_file_get_contents(path.c_str(), &scriptContent, nullptr, nullptr);
```

Comment:
I guess you do not always expect these scripts to be in the datapath? As that should be the case for a stack where datapath is filled could we (like for the extension path) make these relative to the datapath when datapath is set (or additionally check if the scriptpath is a relative path, does not start with a /, and only then append it the the datapath)?